qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -1,5 +1,105 @@
+from functools import partial
+import numpy as np
+
 from qcodes import VisaInstrument
-from qcodes.utils.validators import Numbers, Ints, Enum, MultiType
+from qcodes.instrument.parameter import ArrayParameter
+from qcodes.utils.validators import Numbers, Ints, Enum, Strings
+
+
+class ChannelBuffer(ArrayParameter):
+    """
+    Parameter class for the two channel buffers
+
+    Currently always returns the entire buffer
+    TODO (WilliamHPNielsen): Make it possible to query parts of the buffer.
+        The instrument natively supports this in its TRCL call.
+    """
+
+    def __init__(self, name: str, instrument: SR830, channel: int):
```

Comment:
You might need to do quote i.e. do 'SR830' as a forward declaration because it's defined below the ChannelBuffer and not available when the class is defined